spec/parsers/maven_spec.rb

```diff
@@ -308,23 +310,27 @@
     expect(described_class.match?(fixture_path('ivy_reports/invalid_syntax.xml'))).to be_falsey
   end
 
-  it 'uses parent properties during resolve' do
-    parent_props = {"bibliothecary.version"=>"9.9.9"}
-    deps = described_class.parse_pom_manifest(load_fixture('pom.xml'), parent_props)
+  xdescribe 'parent properties' do
+    # TODO: This parent properties feature doesn't seem to be used anywhere
+    # in the codebase. Is it needed?
```

Comment:
I wonder if this spec fails if you set `parent_props={}`, that could clarify whether it's actually doing anything?